samples/excel/85-preview-apis/range-merged-areas.yaml

```
@@ -0,0 +1,120 @@
+order: 7
+id: excel-merged-areas
+name: Merged areas
+description: This sample shows how to create and then find the merged areas in a worksheet.
+host: EXCEL
+api_set:
+    ExcelAPI: '1.13'
+script:
+    content: |
+      $("#setup").click(() => tryCatch(setup));
+      $("#get-merged-areas").click(() => tryCatch(getMergedAreas));
+
+      async function getMergedAreas() {
+        await Excel.run(async (context) => {
+          // Retrieve the worksheet and the table in that worksheet.
+          const sheet = context.workbook.worksheets.getActiveWorksheet();
+          const table = sheet.getRange("B2:E6");
+          
+          // Retrieve the merged areas within the table and load their details. 
+          let mergedAreas = table.getMergedAreasOrNullObject();
+          mergedAreas.load("address");
+          mergedAreas.load("cellCount");
+
+          // For visibility, highlight the merged area located at cell B2. 
```

Consider using the term "select". In reading "highlight" I was expecting a yellow background to be applied (which you could also do).
```suggestion
          // For visibility, select the merged area. 
```